net-news/alligator/alligator-23.04.49.9999.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_TEST="forceoptional"
+KFMIN=5.75.0
+QTMIN=5.15.0
```

Comment:
Minor: This is not up to same level as other KDE Gear packages for both vars, but would be fixed automatically in future.